Tasks/Kvaskov/Task4/Task4Edit2.py

```diff
@@ -0,0 +1,68 @@
+import sys
+User_Input = int(input("enter the number of characters = "))
+s = 1
+s1 = 0
+b = 0
+d = 0
+d1 = 0
+k = 0
+lenLast = 0
+markers = """!,:;.?- - """
+result1 = []
+if User_Input < 35:
+   print(" Your input is not correct")
+   sys.exit()
+with open('Text1.txt', 'r', encoding='utf-8') as f:
+        LenOfFile = len(f.read())
```

Comment:
this is very bad, you open the file and read the whole content just to check its length, and then you would open it again to go through it "optimal"